sling/nlp/ner/annotators.cc

```diff
@@ -935,6 +980,125 @@ void DateAnnotator::Annotate(const PhraseTable &aliases, SpanChart *chart) {
   }
 }
 
+void AbbreviationAnnotator::Init() {
+  static const char *skipwords[] = {
+    "-", "&",
+    "of", "for", "and", "the", "in",
+    "ltd", "inc", "corp",
```

Comment:
also, "llc" and "gmbh"

Reply:
I have added these, but I think I will need to make all the word lists in the annotators language dependent so they can be configured in languages.sling. For example, for Danish we would like to add A/S, Aps, I/S, K/S, etc., but some of these may be too ambiguous for some languages.